.eslintrc.js

```diff
@@ -28,8 +28,9 @@ module.exports = {
       'warn',
       'always',
       {
-        ignorePattern: 'pragma|ignored',
+        ignorePattern: '.*:|pragma|ignored|prettier',
```

Comment:
We use commented out `key: value` pairs to indicate config options. We don't want those keys capitalised, hence `.*:`. We also don't want to capitalise `// prettier-ignore` comments.